### DISCUSSION TITLE
media object classes removed?

### DISCUSSION THREAD

**Discussion by JacobLett** [twbs/bootstrap#31425] [General] [answered]
Just wondering are the media object classes removed in 5? If so, could this be added to the migration guide in the docs.

**Comment by Deckluhm** [comment, marked as answer]
Duplicate of https://github.com/twbs/bootstrap/discussions/31406, it's already in the migration guide right here: https://v5.getbootstrap.com/docs/5.0/migration/#grid-and-layout

**Reply by JacobLett** [reply]
Thank you!
